fs/accounting/stats.go

```diff
@@ -348,6 +348,8 @@ func (s *StatsInfo) String() string {
 // Transferred returns list of all completed transfers including checked and
 // failed ones.
 func (s *StatsInfo) Transferred() []TransferSnapshot {
+	s.mu.Lock()
```

Comment:
Data is only read in this function, so this should be `s.mu.RLock()` / `s.mu.RUnlock()`. This makes deadlocks less likely.

Reply:
Right. Fixed